### DISCUSSION TITLE
Kathara wipe with no confirmation

### DISCUSSION THREAD

**Discussion by P1c1s** [Kidel/Kathara#327] [Ideas]
Hi everyone, I have a suggest about kathara wipe command. At the moment kathara wants confirmation for deletion of all Kathara devices. But in some cases it might be convenient not to ask for confirmation, maybe for bash script ...
So I think it would be useful to add the -y parameter to the Kathara wipe command such as `kathara wipe -y` or  `kathara wipe y`.

**Comment by Skazza94** [comment]
Hi @P1c1s,
this option is already available in `kathara wipe`.

If you enter `kathara wipe -f`, the `-f` (`--force`) argument disables the confirmation prompt.
